src/components/timecard/start-finish-time-input/StartFinishTimeInput.jsx

```diff
@@ -8,6 +8,9 @@ const StartFinishTimeInput = ({
   errors,
   startTimeValue,
   finishTimeValue,
+  getValues,
```

Comment:
this `getValues` name is passed down all the way through to the `ValidateTimeEntry` component and from the context of this component and the child ones `getValues` doesn't provide much meaning. 
I think the prop should be named to something that tells us what kind of values it's getting. e.g. `getFormValues` (and apply to child components too).

Reply:
updated to getFormValues